### DISCUSSION TITLE
Needed to flash factory firmware before the 1.3.4 release

### DISCUSSION THREAD

**Discussion by IrishBrewer** [Jyers/Marlin#1148] [General Discussion]
Just a heads up in case anyone else runs into this problem.  

I just put together a new Ender 3 V2 I bought off of Amazon and when setting these up, I just go straight to the latest Jyers firmware but in doing so, I end up with a black screen and can't do anything with it.  So, I downloaded the current factory firmware from Creality (https://forums.creality3dofficial.com/download/ender-series/ender-3-v2/) and updated it along with the screen firmware and it booted right up as expected.  After doing this, it accepted the Jyers 1.3.4 firmware readily.

**Comment by Cal-LF** [comment]
I have read that others have done the same steps in order for Jyres FW to be successfully installed. Seems odd.

**Comment by kted** [comment]
The filename of the firmware file has to be different from the last one flashed.
Something went wrong with flashing the 1.3.4, and then no matter how many times you try to flash again without changing the filaname on the microSD, the printer just ignores it.
So, in essence, what you did, was change the filename and it worked.
